sdk/core/core-auth/package.json

```diff
@@ -79,7 +79,6 @@
     "eslint": "^8.0.0",
     "inherits": "^2.0.3",
     "mocha": "^10.0.0",
-    "mocha-junit-reporter": "^2.0.0",
```

Comment:
this must be a dev dep in almost every package, right? why aren't we removing all?

Reply:
Removing the rest will be in another commit to be merged directly without triggering every pipeline.